src/main/java/seedu/address/model/student/TutorialId.java

```diff
@@ -10,7 +10,7 @@
  * Represents an abstract TutorialClass.
  * A TutorialClass can either be an existing class with a specific class code or a "None" type.
  */
-public abstract class TutorialClass {
+public abstract class TutorialId {
 
```

Comment:
Good to see that TutorialClass is not changed to TutorialId which is more readable and makes more sense!